ratelimit/decorators.py

```diff
@@ -20,7 +20,7 @@ def _wrapped(request, *args, **kw):
                                          increment=True)
             request.limited = ratelimited or old_limited
             if ratelimited and block:
-                raise Ratelimited()
+                raise Ratelimited(group=group, key=key, rate=rate)
```

Comment:
Why stop there? How about allow users to give a message for Ratelimit (helpful if you've got a middleware that can't differentiate views).